voyager_archive/views_utils.py

```diff
@@ -1,14 +1,27 @@
 from django.shortcuts import get_object_or_404
+from pymarc import Record
+from typing import List
```

Comment:
This import is not needed, as we're just using `list` (the built-in mutable sequence) and currently just enabling type hints in the IDE.  

`typing.List` is needed for type aliases or run-time type checking.  For example, we could check that `Record.get_fields()` was returning a list of dictionaries, possibly with `List[Dict]` (also importing `typing.Dict`).  We're not doing that, so this import is not needed.
[Documentation if interested](https://docs.python.org/3/library/typing.html)

Confirmed by commenting out the import and running the application.